rails-build/Dockerfile-3.4.2

```diff
@@ -0,0 +1,15 @@
+FROM ruby:3.4.1-slim-bookworm
+
+RUN apt-get update -q \
+    && apt-get install --assume-yes -q --no-install-recommends \
+      curl build-essential git gpg libyaml-dev libffi-dev libpq-dev software-properties-common rsync \
```

Comment:
Note that `libyaml-dev` and `libffi-dev` were necessary additions. [Following this thread](https://github.com/rails/rails/issues/54588)